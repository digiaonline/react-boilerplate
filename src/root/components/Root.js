// @flow

import React from 'react'
import {BrowserRouter as Router, Route} from 'react-router-dom'
import Hello from '../../hello/components/Hello'
import css from './Root.css'

const Root = () => (
  <div className={css.component}>
    <Router>
      <div className={css.routes}>
        <Route exact path="/" component={Hello} />
      </div>
    </Router>
  </div>
)

export default Root
