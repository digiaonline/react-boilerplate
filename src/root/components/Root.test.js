import React from 'react'
import {render} from 'react-dom'
import Root from './Root'

test('Root component renders without crashing', () => {
  const div = document.createElement('div')
  render(<Root />, div)
})
