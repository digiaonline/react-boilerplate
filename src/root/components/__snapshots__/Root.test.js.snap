// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component matches snapshot 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <h1
        className={undefined}
      >
        Hello from Redux.
      </h1>
    </div>
  </div>
</div>
`;
